backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/rest/security/JwtAuthorizationFilter.kt

```diff
@@ -23,13 +24,17 @@ class JwtAuthorizationFilter(authManager: AuthenticationManager, private val jwt
             return
         }
 
-        val user = JWT.require(Algorithm.HMAC512(jwtSecuritySecret.toByteArray()))
-            .build()
-            .verify(header.replace("Bearer ", ""))
-            .subject
+        try {
+            val user = JWT.require(Algorithm.HMAC512(jwtSecuritySecret.toByteArray()))
+                    .build()
+                    .verify(header.replace("Bearer ", ""))
+                    .subject
 
-        SecurityContextHolder.getContext().authentication =
-            if (user != null) JwtAuthenticationToken(user, header) else null
+            SecurityContextHolder.getContext().authentication =
+                    if (user != null) JwtAuthenticationToken(user, header) else null
+        } catch (exception: SignatureVerificationException) {
+            // Token's signature invalid when verified using server-side secret and HMAC512"
```

Comment:
Shouldn't we do something here? What happens if the `SignatureVerificationException` is caught?

Reply:
The catch is not strictly needed, but maybe a log message could go there? In the case of a badly signed token, the filter mirrors the behaviour, as when the Authorization header is absent, namely `chain.doFilter(req, res)` then `return`, user authentication will fail as expected.

Reply:
Okay, thanks :)